lib/inputs/dovetail-downloads.js

```diff
@@ -76,13 +77,13 @@ module.exports = class DovetailDownloads {
         // request data
         remote_referrer: record.remoteReferrer,
         remote_agent: record.remoteAgent,
-        remote_ip: info.geo.masked,
+        remote_ip: iputil.mask(iputil.clean(record.remoteIp)),
```

Comment:
This is the same logic that used to be in `geolocate.js`.  (And I added a test for this).

Reply:
+1 nice to see the ip address here and the calls to clean/mask